_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'Bitcoin Optech Newsletter #127'
+permalink: /en/newsletters/2020/12/09/
+name: 2020-12-09-newsletter
+slug: 2020-12-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Bech32 addresses for taproot and beyond:** Pieter Wuille
+  [published][wuille post] the results of research he conducted with
+  Gregory Maxwell into finding an optimal constant for updated [bech32
+  addresses][topic bech32] to both eliminate the [mutability problem][]
+  and mitigate other data entry problems as best as possible.  He also
+  replied to research performed by several individuals ([1][russell
+  test], [2][casatta test], [3][schmidt test1], [4][schmidt test2]) into
+  which wallets do and don't support [forward address compatibility][],
```

Comment:
I'd suggest breaking this into two sentences and removing the "do and don't":

"He also replied to research performed by several individuals (1, 2, 3, 4) into which wallets support forward address compatibility. That research revealed most wallets will need to be updated in order to pay taproot addresses even if we continue using an unchanged BIP173 address scheme."